interop-tests/src/test/scala/akka/grpc/scaladsl/TelemetrySpec.scala

```diff
@@ -37,14 +40,16 @@ class TelemetrySpec
       val (prefix, method, request) = spi.requests(0)
       prefix should be(GreeterService.name)
       method should be("SayHello")
-      request.entity.contentType should be(GrpcProtocolNative.contentType)
+      request.entity.getContentType should be(GrpcProtocolNative.contentType)
     }
   }
 }
 
 class CollectingTelemetrySpi extends TelemetrySpi {
   var requests: List[(String, String, HttpRequest)] = Nil
```

Comment:
@volatile perhaps if it should be safe to observe from another thread

Reply:
this is just the spec, but yeah, good to add